add index page for docs (#14)
mirumee/saleor-sdk-python: README.md

saleor-sdk tools encode-id User 22
```

## Development

To contribute to this repository you will need Hatch to setup a local development environment. 

Install [Hatch](https://hatch.pypa.io/latest/install/#pipx).

### Documentation

You can deploy a local documentation service, it reloads changes and allow for a live preview of how the documentation will look like after publication:

```
hatch run docs:serve
```

and navigate to http://127.0.0.1:8000

### Tests

To run tests suite use the following command: 

```
hatch run test 
```

### Code style and linters

Use the following commands to format the code and lint it for issues:

```
hatch run lint:fmt
hatch run lint:all
```

### Build and deploy

> This is done by a CI/CD workflow upon the creation of a release but in case of the need for a manual publication of the packager to PyPI you need to build the package archive and publish it. 

Reviewer: ```suggestion
To publish a new version to PyPI, update it's version number in the `pyproject.toml` file and create new github release.

In case when you need to make a new release without GitHub workflow, use following `hatch` commands:
```

## Development

To contribute to this repository you will need Hatch to setup a local development environment. 

Install [Hatch](https://hatch.pypa.io/latest/install/#pipx).

### Documentation

> You can deploy a local documentation service, it reloads changes and allow for a live preview of how the documentation will look like after publication:

Reviewer: This quite a busy sentence. Lets split it into two:

1. Run the blow command to start a dev server with the documentation site:
2. (below the code) Dev server provides a live reload on changes and lets you preview the site after it's published.

### Tests

> With the following command you can run tests: 

Reviewer: ```suggestion
To run tests suite use the following command: 
```

```
hatch run test 
```

> ### Code style

Reviewer: ```suggestion
### Code style and linters
```

> This library follows a specific style guide, to achieve the proper format and lint the code the following commands should be used:

Reviewer: ```suggestion
Use the following commands to format the code and lint it for issues:
```

```
hatch run lint:fmt
hatch run lint:all
```

### Build and deploy

> This is done by a CI/CD workflow upon the creation of a release but in case of the need for a manual publication of the packager to PyPI you need to build the package archive and publish it. 

Reviewer: Is this for the docs or library?

Author: hatch build and publish are for the lib
